medium-tests/src/test/java/mediumtest/fixtures/ServerFixture.java

```diff
@@ -1014,6 +1015,18 @@ private void registerComponentsSearchApiResponses() {
           .build()))));
     }
 
+    private void registerComponentsShowApiResponses() {
+      projectsByProjectKey.forEach((projectKey, project) -> {
+        var url = "/api/components/show.protobuf?component=" + projectKey;
+        var projectComponent = projectsByProjectKey.entrySet().stream().filter(e -> e.getKey().equals(projectKey)).map(entry -> Components.Component.newBuilder().setKey(entry.getKey()).setName(entry.getValue().name).build()).findFirst().get();
+        mockServer.stubFor(get(url)
+          .willReturn(aResponse().withResponseBody(protobufBody(Components.ShowWsResponse.newBuilder()
+            .setComponent(projectComponent)
+            .setPaging(Common.Paging.newBuilder().setTotal(project.relativeFilePaths.size()).build())
```

Comment:
Is this intentional? I think the response is not related to files

Reply:
not intentional, fixed in [29c62f5](https://github.com/SonarSource/sonarlint-core/pull/911/commits/29c62f5cc34d26ba05bb75249c0a79a05ed3d03b)